src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/EntryPointFactoryInterface.php

```diff
@@ -26,5 +26,5 @@ interface EntryPointFactoryInterface
      * This does not mean that the entry point is also used. This is managed
      * by the "entry_point" firewall setting.
      */
-    public function registerEntryPoint(ContainerBuilder $container, string $id, array $config): ?string;
+    public function registerEntryPoint(ContainerBuilder $container, string $firewallName, array $config): ?string;
```

Comment:
This renaming seems unrelated to the new feature. Should we merge it in older branches too, to clean things for named arguments ?

Reply:
This renaming has been done in 5.2: https://github.com/symfony/symfony/pull/37942 (except from the new system - which has done it since 5.1) So I think it's fine to merge in 5.2

Reply:
Yea, it was definitely an "add-on". Sorry, but I knew the idea had been approved and I think it was just missed (?) in #37942 because there were so many.